docs/PyPiPublishing.md

```diff
@@ -0,0 +1,159 @@
+| [Sparsity](https://github.com/openvinotoolkit/nncf/blob/develop/docs/usage/training_time_compression/other_algorithms/Sparsity.md)                                                            | Supported    | Supported     |
+| [Filter pruning](https://github.com/openvinotoolkit/nncf/blob/develop/docs/usage/training_time_compression/other_algorithms/Pruning.md)                                                       | Supported    | Supported     |
+| [Movement pruning](https://github.com/openvinotoolkit/nncf/blob/develop/nncf/experimental/torch/sparsity/movement/MovementSparsity.md)                                                        | Experimental | Not supported |
+
+- Automatic, configurable model graph transformation to obtain the compressed
+  model.
+  > **NOTE**: Limited support for TensorFlow models. Only models created using
+    Sequential or Keras Functional API are supported.
+- Common interface for compression methods.
+- GPU-accelerated layers for faster compressed model fine-tuning.
+- Distributed training support.
+- Git patch for prominent third-party repository
+  ([huggingface-transformers](https://github.com/huggingface/transformers))
+  demonstrating the process of integrating NNCF into custom training pipelines.
+- Seamless combination of pruning, sparsity, and quantization algorithms. Refer
+  to [optimum-intel](https://github.com/huggingface/optimum-intel/tree/main/examples/openvino)
+  for examples of joint (movement) pruning, quantization, and distillation
+  (JPQD), end-to-end from NNCF optimization to compressed OpenVINO IR.
+- Exporting PyTorch compressed models to ONNX\* checkpoints and TensorFlow
+  compressed models to SavedModel or Frozen Graph format, ready to use with
+  [OpenVINO&trade; toolkit](https://docs.openvino.ai).
+- Support for [Accuracy-Aware model training](https://github.com/openvinotoolkit/nncf/blob/develop/docs/usage/training_time_compression/other_algorithms/Usage.md#accuracy-aware-model-training)
+  pipelines via the [Adaptive Compression Level Training](https://github.com/openvinotoolkit/nncf/blob/develop/docs/accuracy_aware_model_training/AdaptiveCompressionLevelTraining.md)
+  and [Early Exit Training](https://github.com/openvinotoolkit/nncf/blob/develop/docs/accuracy_aware_model_training/EarlyExitTraining.md).
+
+
+
+## Installation Guide
+
+NNCF can be installed as a regular PyPI package:
+
+```bash
+pip install nncf
+```
+
+For detailed installation instructions, refer to the
+[Installation](https://github.com/openvinotoolkit/nncf/blob/develop/docs/Installation.md) guide.
+
+
+
+### System Requirements
+
+- Ubuntu 18.04 or later (64-bit)
+- Python 3.8 or later
+- Supported frameworks:
+  - PyTorch >=2.2, <2.4
+  - TensorFlow >=2.8.4, <=2.15.1
+  - ONNX ==1.16.0
+  - OpenVINO >=2022.3.0
```

Comment:
@MaximProshin, we would need to support this and change it for every release along with the `Readme.md`.

Reply:
Is this information updated in the readme file for every release? We can either use this information or we can refer to the readme from the pypi page.

Reply:
It's not updated every release, but yes, we will need to make sure it's up to date every time